sales_team_operating_unit/models/crm_team.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2017-TODAY Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
```

Comment:
@lreficent Sorry my mistake it has to removed from test cases but instead we removed from crm_team.py

So you need to add the import again to py file and remove it from here.